InferenceSystem/src/LiveInferenceOrchestrator.py

```diff
@@ -36,8 +36,10 @@
 ORCASOUND_LAB_LOCATION = {"id": "rpi_orcasound_lab", "name": "Haro Strait", "longitude":  -123.17357, "latitude": 48.55833}
 PORT_TOWNSEND_LOCATION = {"id": "rpi_port_townsend", "name": "Port Townsend", "longitude":  -122.76045, "latitude": 48.13569}
 BUSH_POINT_LOCATION = {"id": "rpi_bush_point", "name": "Bush Point", "longitude":  -122.6039, "latitude": 48.03371}
+SUNSET_BAY_LOCATION = {"id": "rpi_sunset_bay", "name": "Sunset Bay", "longitude":  -122.3339, "latitude": 47.86497}
```

Comment:
Perhaps a follow up task, but we should aim to have static info like location injected via an environment variable or some kind of config rather than hard coding it in the script.

Reply:
Completely agree, I can take this up